aspnetcore/release-notes/aspnetcore-2.2.md

```diff
@@ -13,7 +13,7 @@ This article highlights the most significant changes in ASP.NET Core 2.2, with l
 
```

Comment:
Note: I'm not updating the markdown headers because the changes primarily address nits.